src/crudOps.js

```diff
@@ -0,0 +1,102 @@
+class Store {
+  // get todo items from local storage
+  static getTodos = () => JSON.parse(localStorage.getItem('todos')) || [];
+
+  // add todo item to local storage
+  static addTodo = (todo) => {
+    const todos = this.getTodos();
+    todos.push(todo);
+
+    localStorage.setItem('todos', JSON.stringify(todos));
+  };
```

Comment:
- [ ] I am not able to add a `to do` that means I am not able to test other methods please address that

Reply:
- [ ] Remember to update your README

Reply:
please get the project started using npm start on ther server

Reply:
readme updated